api-reference/10 UI Components/DateBoxBase/1 Configuration/dateSerializationFormat.md

```diff
@@ -0,0 +1,26 @@
+---
+id: DateBoxBase.Options.dateSerializationFormat
+type: String
+default: undefined
+---
+---
+##### shortDescription
+Specifies the date-time value serialization format. Use it only if you do not specify the [value](/Documentation/ApiReference/UI_Components/dxCalendar/Configuration/#value) at design time.
+
+---
+Without a value, the UI component cannot detect its format. In this case, specify the **dateSerializationFormat** property that supports the following formats:
+
+- `"yyyy-MM-dd"` - a local date  
```

Comment:
` a local date ` Any local date? How is this different from the local date and time below?

Reply:
I'm going to remove this information at all.